core/templatetags/social_tags.py

```diff
@@ -0,0 +1,32 @@
+from django import template
+import urllib.parse
+
+register = template.Library()
+
+
+@register.simple_tag
+def build_twitter_link(text, url):
+    encoded_text = urllib.parse.quote(text + ' ')
+    link = 'https://twitter.com/intent/tweet?text={}{}'
+    return link.format(encoded_text, url)
```

Comment:
please add tests for these tags. examples [here](https://github.com/uktrade/directory-ui-supplier/blob/master/company/tests/test_templatetags.py)